### ISSUE TITLE
nuxt 3 local - signIn path not matching

### ISSUE THREAD

**Issue by manuel-84** [sidebase/nuxt-auth#335]
### Environment

(static website made on nuxt 3)
```
  "devDependencies": {
    "@nuxtjs/i18n": "^8.0.0-beta.10",
    "@nuxtjs/tailwindcss": "^6.6.5",
    "@pinia/nuxt": "^0.4.8",
    "@sidebase/nuxt-auth": "^0.6.0-alpha.1",
    "@vite-pwa/nuxt": "^0.0.7",
    "@vueuse/core": "^9.13.0",
    "nuxt": "^3.3.3",
    "pinia": "^2.0.34",
  }
```

### Reproduction

in nuxt config:
```
  ssr: false,
  auth: {
    provider: {
      type: 'local'
    },
    pages: {
      login: '/'
    },
    baseURL: '....amazonaws.com/uat/api',
    endpoints: {
      signIn: { path: 'users/login', method: 'post' }
    }
  }
```

### Describe the bug

when calling signIn the path called is wrong and also method "post" is not used:

```
curl '........amazonaws.com/uat/api/login' \
  -H 'sec-ch-ua: "Chromium";v="112", "Microsoft Edge";v="112", "Not:A-Brand";v="99"' \
  -H 'DNT: 1' \
  -H 'sec-ch-ua-mobile: ?0' \
  -H 'User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/112.0.0.0 Safari/537.36 Edg/112.0.1722.48' \
  -H 'content-type: application/json' \
  -H 'accept: application/json' \
  -H 'Referer: http://localhost:3000/' \
  -H 'sec-ch-ua-platform: "Windows"' \
  --data-raw $'{"email":"test@xxx.xxx","password":"xxxxx"}' \
  --compressed
```

what is expected here is signIn path to be appended after baseURL

### Additional context

I'm using a login component where code is taken from your example:

```
<script lang="ts" setup>
const { signIn, token, data, status, lastRefreshedAt, signOut, getSession } = useAuth()
const email = ref('')
const password = ref('')
</script>

<template>
  <div>
    <pre>Status: {{ status }}</pre>
    <pre>Data: {{ data || 'no session data present, are you logged in?' }}</pre>
    <pre>Last refreshed at: {{ lastRefreshedAt || 'no refresh happened' }}</pre>
    <pre>JWT token: {{ token || 'no token present, are you logged in?' }}</pre>
    <form @submit.prevent="signIn({ email, password })">
      <input v-model="email" type="text" placeholder="email">
      <input v-model="password" type="password" placeholder="Password">
      <button type="submit">
        sign in
      </button>
    </form>
    <br>
    <button @click="signIn({ email, password }, { callbackUrl: '/' })">
      sign in (with redirect to /)
    </button>
    <br>
    <button @click="signOut({ callbackUrl: '/signout' })">
      sign out
    </button>
    <br>
    <button @click="getSession({ required: false })">
      refresh session (required: false)
    </button>
    <br>
    <button @click="getSession({ required: true, callbackUrl: '/' })">
      refresh session (required: true)
    </button>
    <NuxtPage />
  </div>
</template>
```

**Comment by serman** [comment]
just to be sure: according to this: http://sidebase.io/nuxt-auth/v0.6/configuration/nuxt-config baseURL is a property of root config, not the providers section. 
For me, it is working like in the snippet I pasted above.

**Comment by manuel-84** [comment]
> just to be sure: according to this: http://sidebase.io/nuxt-auth/v0.6/configuration/nuxt-config baseURL is a property of root config, not the providers section. For me, it is working like in the snippet I pasted above.

yes, but for me is not working in any way

and since I also faced an error with `signInResponseTokenPointer` i decided to give up with this package and write my own storage

**Comment by tnt-freskim-veliu** [comment]
This is my auth config in nuxt.config,js, and currently it's working except for the signOut because it's not sending the bearer token in the backend:

``

auth: {
        provider: {
            type: 'local',
            endpoints: {
                signIn: {path: '/login', method: 'post'},
                signUp: {path: '/register', method: 'post'},
                signOut: {path: '/logout', method: 'post'},
                getSession: {path: "/profile", method: "get"},
            },

            token: {
                signInResponseTokenPointer: '/data/token',
                type: 'Bearer',
                headerName: 'Authorization'
            }
        },

        pages: {
            login: '/login'
        },
        baseURL: process.env.API_BASE_URL,

        session: {
            enableRefreshOnWindowFocus: false
        }
    },

``

**Comment by BracketJohn** [comment]
Hey!

Just did a quick test and I can confirm that if you change the `path` inside sign-in the composable also uses it for trying to sign in:
![image](https://user-images.githubusercontent.com/25911230/233807090-5cc0fb09-c31b-4b49-b95e-60dddb43ed73.png)

@manuel-84 we need a reproduction repository from you to vberify this further!

> except for the signOut because

ah, good one! I fixed this in #347 -> the token is now also sent on sign out (: 

I'll do a release in the next hour or so that should adress this (:

**Comment by BracketJohn** [comment]
closing as reproduction is missing - feel free to reopen with a repro, thanks!
